airflow/dags/amplitude_benefits/jsonl_bigquery.sql

```diff
@@ -67,6 +67,6 @@ CREATE OR REPLACE EXTERNAL TABLE `amplitude.benefits_events` (
 )
 OPTIONS (
     format = "JSON",
-    uris = ["{{get_bucket()}}/amplitude/benefits/*.jsonl"],
+    uris = ["{{'gs://ingest_amplitude_raw_dev' if get_bucket() == 'gs://gtfs-data-test' else 'gs://ingest_amplitude_raw_prod'}}/amplitude/benefits/*.jsonl"],
```

Comment:
I would prefer to have the code here use [`is_development()`](https://github.com/cal-itp/calitp-py/blob/435bc3ebdfb0bc69f6240286b959ba24a85096a9/calitp/config.py#L17) from calitp-py instead of `if get_bucket() == 'gs://gtfs-data-test'` - as this is quite brittle and would break if `gtfs-data-test` name gets changed.

This is what I would like to change the code to:

```diff
CREATE OR REPLACE EXTERNAL TABLE `amplitude.benefits_events` (
 )
 OPTIONS (
     format = "JSON",
-    uris = ["{{'gs://ingest_amplitude_raw_dev' if get_bucket() == 'gs://gtfs-data-test' else 'gs://ingest_amplitude_raw_prod'}}/amplitude/benefits/*.jsonl"],
+    uris = ["{{'gs://ingest_amplitude_raw_dev' if is_development() else 'gs://ingest_amplitude_raw_prod'}}/amplitude/benefits/*.jsonl"],
     ignore_unknown_values = True
 )
```

I am unable, however, to import the `is_development` method into this file.

Reply:
Resolved with https://github.com/cal-itp/data-infra/pull/1276/commits/f154a73cb8a28c179574517b652aa2f79938033f